packages/web-components/src/calendar/calendar.stories.ts

```diff
@@ -100,7 +105,7 @@ const CalendarTemplate = ({
     ${yearFormat ? `year-format="${yearFormat}"` : ''}
     ${disabledDates ? `disabled-dates="${disabledDates}"` : ''}
     ${selectedDates ? `selected-dates="${selectedDates}"` : ''}
-    ${readonly ? `readonly` : ''}
+    ${readonly === false ? `readonly="false"` : ''}
```

Comment:
It seems like this is working, but I can't figure out why and I think the negation shouldn't be necessary. If it's a boolean attribute, the existence will mean `true` even if the value provided is `false`. Confusing to me, but that's html.